src/App.tsx

```diff
@@ -1,8 +1,32 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext, useEffect, useRef, useState,
+} from 'react';
+import cn from 'classnames';
+import {
+  createTodos, getTodos, removeTodo, updateTodo,
+} from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Todo } from './types/Todo';
+import { TodoComponent } from './components/Todo';
+import { Footer } from './components/Footer';
+import { ErrorMessage } from './components/ErrorMessage';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [inputTitle, setInputTitle] = useState('');
+  const [isAdding, setIsAdding] = useState<Todo | null>(null);
```

Comment:
from name `isAdding` I expect to see boolean value here, but instead of this I see Todo or null

it's strange